skipruntime-ts/adapters/postgres/src/index.ts

```diff
@@ -206,9 +206,9 @@ export class PostgresExternalService implements ExternalService {
             `
 CREATE OR REPLACE FUNCTION %I() RETURNS TRIGGER AS $f$
 BEGIN
-  IF NEW IS NOT NULL THEN
+  IF TG_OP = 'INSERT' OR TG_OP = 'UPDATE' THEN
```

Comment:
What happens when `TG_OP` is `DELETE` or `TRUNCATE`?

Reply:
'OLD' is set for DELETE triggers; see https://www.postgresql.org/docs/current/plpgsql-trigger.html 

as for TRUNCATE, it won't invoke row-level triggers like this one -- I'll file an issue for now and look into the best way to handle that going forward.  would need another statement-level trigger on the PG side, and some additional logic on the skip side to clear out the collection when that trigger fires.